qa/zcash/performance-measurements.sh

```diff
@@ -65,6 +65,15 @@ function zcashd_valgrind_stop {
     cat valgrind.out
 }
 
+function extract_benchmark_data {
+    if [ ! -f "block-107134.tar.gz" ]; then
+        zcashd_stop
+        echo "block-107134.tar.gz not found."
```

Comment:
Let's figure out hosting/distribution later (outside of this PR). However, for this PR, can we do a sha256sum check so that the hash is in this script (and revision control)? That mght save us some headache in diagnosing issues in the future and ensures we have some revision history for the test vector.